README.md

```diff
@@ -6,80 +6,134 @@ You're working on your Tunr app and you encounter this error. What does it mean
 
 ![Rails error](http://i.imgur.com/9NR7XNT.png)  
 
-```text
-Your answer...
+```
+The first part of an error message identifies what template is missing. The second part part of the message contains a hash and the last part of this message tells us where Rails has looked for the templates.
+The error message here is saying you are missing a template aka you are missing the page artist.index.html.erb
+
+When in doubt. GOOGLE!!!!!
+
 ```
 
 ### Question 2
 
-What is the difference between...  
-• Creating a migration to add a column   
-• Editing an existing migration and running `rake db:migrate:reset`   
+Consider this file name:
+
+```
+20150726145027_create_artists.rb
+```
+
+What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+It is a migration file. It is used to create the artist table in the database. The numbers in the beginning represent when the migration file was created. The timestamp tells the computer in what order to run the migrates.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
-```text
-Your answer...
+```
+Within a route, a controller & action is specified. For example,  and an action   in  "artists#new", the right indicates the artists controller and the ac its specifying that there is a new artists being added. Thus it would go to the artist.new.html.erb.
```

Comment:
the router maps routes (verb + path) to a controller action. "/artists/new" maps to `artists#new`